### DISCUSSION TITLE
Internal forces default axes

### DISCUSSION THREAD

**Discussion by cyclonehopper** [Dlubal-Software/RFEM_Python_Client#347] [Q&A]
Hello Everyone,

I am exploring this API, and what I am trying to do is create the model in GUI, solve the model and  retrieve internal forces and do something about it. It appears the member forces are reported based on z-down configuration even if the model was created in GUI where local member is y-up configuration.  just wanted to know if this the default behavior?  and is there a setting to set from within the API to get the forces based on the local axes setting in the current model being read?

I attached a screenshot of the moment diagram in GUI showing Mz;  but in the Julia REPL (Python API call via PyCall) shows up as My in the  results.

This is my first time posting here.

Regards
Cyclonehopper

![image](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/112364371/ccfdb61a-df54-4b81-bcbe-c7413c90bedb)

**Comment by heetrojivadiya** [comment]
Hello @cyclonehopper, 

I have tested this and I am also getting the same results. It seems like extracting result tables from WS always give default one either setting different local member axis system via WS or manually in RFEM6. But, you may still convert that result table by adding few lines after retrieving result table and you will get list of dictionaries according to you local axis: 
```py
results = ResultTables.MembersInternalForces(object_no=0, include_base=True)

for item in results:
    if 'internal_force_my' in item and 'internal_force_mz' in item:
        item['internal_force_my'], item['internal_force_mz'] = item['internal_force_mz'], item['internal_force_my']
    if 'internal_force_vy' in item and 'internal_force_vz' in item:
        item['internal_force_vy'], item['internal_force_vz'] = item['internal_force_vz'], item['internal_force_vy']

```
![960548cb-a5b1-4a50-afd8-b2061be76a0b](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/108512789/314cd0f5-5c97-429d-9562-b5d11e6944e7)
![155d55b4-f05a-4854-be0a-5192023d6a70](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/108512789/5b042492-6cc5-44d0-8379-6b68d0ed6e84)

Thank you for reaching out to us. We will try to solve this problem as soon as possible.
